service/src/test/java/bio/terra/tanagra/service/accesscontrol/AouWorkbenchAccessControlTest.java

```diff
@@ -295,7 +295,11 @@ private void assertStudyPermissions(UserId user, ResourceId resource, Action...
   private void assertStudyNoListPermissions(UserId user, ResourceType resource, Action... actions) {
     ResourceCollection resources =
         accessControlService.listAuthorizedResources(
-            user, Permissions.forActions(resource, actions), 0, Integer.MAX_VALUE);
+            user,
+            Permissions.forActions(resource, actions),
+            (ResourceId) null,
```

Comment:
The effect is the same, but this should be cast to string iiuc. 
for list underlays: resource id is correct since it is for parent resource. this in turn calls the function with string param
for studies, it would be user group. 
---
If the overloaded functions are too confusing we can look at changing it in a followup PR

Reply:
The compiler complained, so I did the cast to specifically call what it was previously calling :-) Yes maybe in a follow-up PR we can fix this

Reply:
I mean you would need to cast it to string